lisa/util/process.py

```diff
@@ -230,6 +230,17 @@ def wait_result(
         timer = create_timer()
         is_timeout = False
 
+        if (
+            self._sudo
+            and isinstance(self._shell, SshShell)
+            and self._shell._is_sudo_required_password
```

Comment:
`_is_sudo_required_password` is a private variable, so it shouldn't be used outside. Mentioned in another comment, make it public.

Reply:
Done